ebms-send-in/src/main/kotlin/no/nav/emottak/ebms/FagmeldingService.kt

```diff
@@ -121,6 +143,8 @@ object FagmeldingService {
                                 ),
                                 requestId = Uuid.random().toString()
                             ).also {
+                                val refParam = refParam(sendInRequest.asEIFellesFormat())
```

Comment:
Trenger ikke wrappe i fellesformat på nytt. Den er wrappet allerede tidligere i flyten, og kan refereres til direkte

`refParam(this)`